proof/aggregate.saw

```diff
@@ -0,0 +1,398 @@
+/*
+ * Copyright (c) 2020 Galois, Inc.
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+*/
+
+///////////////////////////////////////////////////////////////////////////////
+// Specifications
+///////////////////////////////////////////////////////////////////////////////
+
+let blst_pairing_init_spec DST_len = do {
+  ctx_ptr <- crucible_alloc (llvm_struct "struct.PAIRING");
+  hash_or_encode <- crucible_fresh_var "hash_or_encode" (llvm_int 32);
+  //(DST, DST_ptr) <- ptr_to_fresh_readonly "DST" (llvm_array DST_len (llvm_int 8)); // not strictly needed for this function
```

Comment:
is `DST_ptr` not used?

Reply:
The function copies the pointer to `ctx->DST`, but it does not read what's pointed to.